docs/blog/20230904-Daeraxa-SeptemberUpdate.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Community Update
+author: Daeraxa
+date: 2023-09-04
+category:
+  - news
+  - log
+tag:
+  - update
+---
+
+I want you to act as a Pulsar team member writing a blog post. I will provide titles of recent changes. You will elaborate on each topic and make each paragraph coherent from my ramblings. You will welcome all readers to the September Pulsar community update.
```

Comment:
Might wanna change out this preview, unless it's stylistic, comedic choice lol

Reply:
Lol it was designed to sound like a chat GPT prompt in the funny blerb section I normally do.

Reply:
Lol no no not at all. I genuinely thought it was a leftover prompt you used for help or something. In that case, this is perfect and absolutely gave me a cuckle